easybuild/easyconfigs/c/Cookiecutter/Cookiecutter-1.4.0-goolf-1.7.20-Python-2.7.11.eb

```diff
@@ -0,0 +1,81 @@
+# This file is an EasyBuild reciPY as per https://github.com/hpcugent/easybuild
+# Author: Pablo Escobar Lopez
+# sciCORE - University of Basel
+# SIB Swiss Institute of Bioinformatics 
+
+easyblock = 'Bundle'
+
+name = 'Cookiecutter'
+version = '1.4.0'
+versionsuffix = '-Python-%(pyver)s'
+
+homepage = 'https://github.com/audreyr/cookiecutter'
+description = """Command-line utility that creates projects from cookiecutters (project templates). 
+ E.g. Python package projects, jQuery plugin projects."""
+
+toolchain = {'name': 'goolf', 'version': '1.7.20'}
+
+# this is a bundle of Python packages
+exts_defaultclass = 'PythonPackage'
+
+dependencies = [
+    ('Python', '2.7.11'),
+]
+
+# this application also requires numpy, scipy and matplotlib which 
+# are provided by the Python module 
+exts_list = [
+    ('six', '1.10.0', {
+        'source_urls': ['https://pypi.python.org/packages/source/s/six/'],
+    }),
+    ('python-dateutil', '2.6.0', {
+        'source_urls': ['https://pypi.python.org/packages/source/p/python-dateutil/'],
+	'modulename': 'dateutil',
+    }),
+    ('future', '0.16.0', {
+        'source_urls': ['https://pypi.python.org/packages/source/f/future/'],
+    }),
+    ('whichcraft', '0.4.0', {
+        'source_urls': ['https://pypi.python.org/packages/source/w/whichcraft/'],
+    }),
+    ('arrow', '0.8.0', {
+        'source_urls': ['https://pypi.python.org/packages/source/a/arrow/'],
+    }),
+    ('jinja2-time', '0.2.0', {
+        'source_urls': ['https://pypi.python.org/packages/source/j/jinja2-time/'],
+	'modulename': 'jinja2',
```

Comment:
@pescobar seems like this pulls in `Jinja2`, so it should be listed after the `Jinja2` entry?